### ISSUE TITLE
test-runner in docker custom slugs / challenges

### ISSUE THREAD

**Issue by Daedra22** [exercism/exercism#6431]
Hi guys,
I would like to ask, if it is possible to create own slugs / challenges locally and test them with the exercism test-runners ?

As far as I see, you need to pass the key of the excercise slug, the solution code and the output dir for the results.json to the test-runner scripts:
`./bin/run-in-docker.sh <exercise-slug> <solution-dir> <output-dir>`

But I would like to create locally the tests (exercise-slug) and use the runner to test my solution using docker.

Is it possible?

**Comment by glennj** [comment]
Certainly. You need to install Docker on your machine. For Ubuntu, for example, `apt install docker.io` -- https://packages.ubuntu.com/jammy/docker.io

**Comment by Daedra22** [comment]
> Certainly. You need to install Docker on your machine. For Ubuntu, for example, `apt install docker.io` -- https://packages.ubuntu.com/jammy/docker.io

Hi, thanks for the reply.
I have docker installed.
But my question was about whether I can execute own challenges, own slugs.
Currently I have to pass the slug name and the container takes the tests of slugs from the exercism website. But i would like the runner to execute my own tests (slugs).

**Comment by glennj** [comment]
What test runner are you using? Whether it talks to the exercism website is simply programmed into it. The test runners I've written don't do that.
